content/cif-connector/src/main/content/jcr_root/libs/dam/content/schemaeditors/forms/default/image/items/tabs/items/tab4/.content.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="nt:unstructured"
+    jcr:title="Commerce"
+    sling:resourceType="granite/ui/components/coral/foundation/container"
+    granite:rel="aem-assets-metadata-form-tab"
+    granite:class="cq-dam-asset-metadata-commerce"
+    listOrder="3">
+    <items jcr:primaryType="nt:unstructured">
+        <col1
```

Comment:
Is it best practice to add col1 & col3 as empty columns? Why not just put the product field in column 1?

Reply:
Indeed, that seems to be useless, although in the metadata editor they prefer to have the 3 columns even if they are empty.

Reply:
@mhaack , BTW, the 3 columns are also related to the metadata schema editor, which adds the empty columns by default when you add a new tab. So, it's better to keep the empty columns.

Reply:
Ok that makes sense than.